internal/install/recipe_installer.go

```diff
@@ -102,16 +110,35 @@ func (i *recipeInstaller) install() {
 		}
 	}
 
-	log.Infoln("Success! Your data is available in New Relic.")
-	log.Infoln("Go to New Relic to confirm and start exploring your data.")
+	profile := credentials.DefaultProfile()
+	fmt.Printf(`
+	Success! Your data is available in New Relic.
+
+	Go to New Relic to confirm and start exploring your data.
+	https://one.newrelic.com/launcher/nrai.launcher?platform[accountId]=%d
+	`, profile.AccountID)
+
+	fmt.Println()
 }
 
 func (i *recipeInstaller) discoverFatal() *discoveryManifest {
+	s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
```

Comment:
We can consider abstracting this later on maybe?

Reply:
Yeah, I think to get a standard, we should probably move that out to a method.